### PR TITLE
Bug 1908169: The example of Import URL is "Fedora cloud image list" for all templates.

### PR DESCRIPTION
**Pull request by avivtur**
**Fixes**: 
https://bugzilla.redhat.com/show_bug.cgi?id=1908169

**Analysis / Root cause**: 
The example of OS image in add-boot-source-form was unclear since it is possible to take any template in U/S and get fedora link image example and  same in D/S you with RHEL image link.

**Solution Description**: 
changing the behavior to adjust message and image link accordingly to OS and not according U/S or D/S.
NOTE: in case of OS is not RHEL or Windows, to default will be a link to Fedora image.

**Screen shots / Gifs for design review**: 

Before: 
---Downstream---
bz_1908169_DS_before

---Upstream---
bz_1908169_US_before

After:
---Downstream---
bz_1908169_DS_after_3

bz_1908169_DS_after_4

bz_1908169_DS_after_3

bz_1908169_DS_after_2

---Upstream---
bz_1908169_US_after

Signed-off-by: Aviv Turgeman <aturgema@redhat.com>

### CODE REVIEW

**Comment by avivtur** [comment]
@yaacov please review

**Comment by yaacov** [comment]
/ok-to-test

a - need to add the `baseImageName` in the wizard and in the regular add disk modal
b - if it's a none required prop, it can to `.include` without checking it's not `null`, but as for point a, you can just make it required :-)

File: frontend/packages/kubevirt-plugin/src/components/form/helper/url-source-help.tsx [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
-export const URLSourceHelp: React.FC = () => {
+type URLSourceHelpProps = {
+  baseImageName: string;
+};
+
+export const URLSourceHelp: React.FC<URLSourceHelpProps> = ({ baseImageName }) => {
   const { t } = useTranslation();
-  const isUpstream = window.SERVER_FLAGS.branding === 'okd';
+  // checking os is RHEL/Windows and adjust link images accordingly, Fedora is default for all other OS.
+  const body = baseImageName.includes('rhel') ? (
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/form/helper/url-source-help.tsx:13]:
```suggestion
  const body = baseImageName?.includes('rhel') ? (
```

File: frontend/packages/kubevirt-plugin/src/components/form/helper/url-source-help.tsx [hunk 2]
```diff
@@ -1,34 +1,49 @@
...
+    <Trans t={t} ns="kubevirt-plugin">
+      Example: For RHEL, visit the{' '}
+      <strong>
+        <a href={RHEL_IMAGE_LINK} rel="noopener noreferrer" target="_blank">
+          RHEL download page
+        </a>
+      </strong>{' '}
+      (requires login) and copy the download link URL of the KVM guest image
+    </Trans>
+  ) : baseImageName.includes('win') ? (
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/form/helper/url-source-help.tsx:23]:
```suggestion
  ) : baseImageName?.includes('win') ? (
```

**Pull request merged**